clients/fossid-webapp/src/main/kotlin/model/status/DownloadStatus.kt

```diff
@@ -24,8 +24,10 @@ import com.fasterxml.jackson.annotation.JsonProperty
 enum class DownloadStatus {
     FAILED,
     FINISHED,
+
```

Comment:
The annotation below is also for serialization and the enum value is undocumented, but the annotation remains on a separate line. This seems to be inconsistent. Is there any hidden rule there I'm missing?